### DISCUSSION TITLE
External Material Model Error

### DISCUSSION THREAD

**Discussion by MohammadAlturki** [Anura3D/Anura3D_OpenSource#61] [Q&A]
Hi all,

I am trying to use an external material model for the soil. I am sure about doing everything the way it should be done. However, an error is encountered when I try to run my simulation, as shown in the attached figure. 

Thank you
![Screenshot 2023-01-30 134620](https://user-images.githubusercontent.com/123959668/215494470-61fe50c7-8a6f-4046-a906-b7c814342aa9.png)

**Comment by murphy-john** [comment]
Will you provide additional information about what you've done so far? It's hard to help with such little information. It appears the file for the external soil model is not in the working directory. 

Please read our [posting guidelines](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29#discussion-4274028). They will help you identify the issue yourself and post necessary details in order to help you.

**Reply by MohammadAlturki** [reply]
Hi John, thanks for your reply.
Please find additional information in the attached pdf file.

[Soil model problem.pdf](https://github.com/Anura3D/Anura3D_OpenSource/files/10677402/Soil.model.problem.pdf)
